sdk/servicebus/service-bus/test/stress/scenarioBatchReceive.ts

```diff
@@ -45,7 +45,7 @@ function sanitizeOptions(args: string[]): Required<ScenarioReceiveBatchOptions>
     numberOfMessagesPerSend: options.numberOfMessagesPerSend || 1,
     delayBetweenSendsInMs: options.delayBetweenSendsInMs || 0,
     totalNumberOfMessagesToSend: options.totalNumberOfMessagesToSend || Infinity,
-    sendAllMessagesBeforeReceiveStarts: options.sendAllMessagesBeforeReceiveStarts,
+    sendAllMessagesBeforeReceiveStarts: !!options.sendAllMessagesBeforeReceiveStarts,
```

Comment:
Why?
Did the default at line 37 not work?

Reply:
The property for that field is still marked as optional by the time we get here. I'm not sure which piece is supposed to flip that to Required<>?

Reply:
I see, this is fine for the PR. I can check later.